clients/git/client_test.go

```diff
@@ -88,20 +89,13 @@ func TestInitRepo(t *testing.T) {
 	}{
 		{
 			name:        "Success",
-			uri:         "file://%s",
+			uri:         "%s",
```

Comment:
was there a problem with the `file://` scheme? Does it have to do with needing to call `MakeLocalDirRepo`?

Reply:
It doesn't.

Reply:
The ` localdir.MakeLocalDirRepo(uri)` doesn't handle it.

Reply:
👍

Reply:
uri could probably just be removed at this point if all the test cases just use %s